package.json

```diff
@@ -9,12 +9,14 @@
   },
   "scripts": {
     "start": "node service.js",
-    "lint": "./node_modules/.bin/semistandard *.js config/*.js",
-    "test": "npm run lint"
+   	"lint": "./node_modules/.bin/semistandard *.js config/*.js",
+    "test": "bash -c 'source ./config/development.env; ./node_modules/.bin/lab --ignore __core-js_shared__ --flat --threshold 68 -r html -o ./coverage/coverage.html -r lcov -o ./coverage/lcov.info -r json -o ./coverage/coverage.json -r console -o stdout'",
```

Comment:
add linting to 'npm test' (we do this in other services), e.g. 

    "test": "npm run lint && bash -c ....